core/src/main/java/com/statful/client/core/message/MessageBuilder.java

```diff
@@ -24,6 +24,7 @@
     private String value;
     private long timestamp;
     private AggregationFrequency aggregationFrequency = AggregationFrequency.FREQ_10;
+    private int sampleRate;
```

Comment:
This can behave as a default of 0 which is strange. Possibly set the default value (100), change to Integer and use a null or Optional?